engine/swarm/configs.md

```diff
@@ -0,0 +1,517 @@
+mounted into `C:\ProgramData\Docker\configs` and symbolic links are created to
+the desired location, which defaults to `C:\<config-name>`.
+
+You can update a service to grant it access to additional configs or revoke its
+access to a given config at any time.
+
+A node only has access to configs if the node is a swarm manager or if it is
+running service tasks which have been granted access to the config. When a
+container task stops running, the configs shared to it are unmounted from the
+in-memory filesystem for that container and flushed from the node's memory.
+
+If a node loses connectivity to the swarm while it is running a task container
+with access to a config, the task container still has access to its configs, but
+cannot receive updates until the node reconnects to the swarm.
+
+You can add or inspect an individual config at any time, or list all
+configs. You cannot remove a config that a running service is
+using. See [Rotate a config](configs.md#example-rotate-a-config) for a way to
+remove a config without disrupting running services.
+
+In order to update or roll back configs more easily, consider adding a version
+number or date to the config name. This is made easier by the ability to control
+the mount point of the config within a given container.
+
+## Read more about `docker config` commands
+
+Use these links to read about specific commands, or continue to the
+[example about using configs with a service](configs.md#example-use-configs-with-a-service).
+
+- [`docker config create`](/engine/reference/commandline/config_create.md)
+- [`docker config inspect`](/engine/reference/commandline/config_inspect.md)
+- [`docker config ls`](/engine/reference/commandline/config_ls.md)
+- [`docker config rm`](/engine/reference/commandline/config_rm.md)
+
+## Examples
+
+This section includes graduated examples which illustrate how to use
+Docker configs.
+
+> **Note**: These examples use a single-Engine swarm and unscaled services for
+> simplicity. The examples use Linux containers, but Windows containers also
+> support configs.
+
+### Simple example: Get started with configs
+
+This simple example shows how configs work in just a few commands. For a
+real-world example, continue to
+[Intermediate example: Use configs with a Nginx service](#advanced-example-use-configs-with-a-nginx-service).
+
+1.  Add a config to Docker. The `docker config create` command reads standard
+    input because the last argument, which represents the file to read the
+    config from, is set to `-`.
+
+    ```bash
+    $ echo "This is a config" | docker config create my-config -
+    ```
+
+2.  Create a `redis` service and grant it access to the config. By default,
+    the container can access the config at `/my-config`, but
+    you can customize the file name on the container using the `target` option.
+
+    ```bash
+    $ docker service  create --name="redis" --config="my-config" redis:alpine
```

Comment:
These flags don't need double quotes. They aren't harmful, but stylistically it's out of the ordinary.